deploy/deploy_full.sh

```diff
@@ -14,12 +14,12 @@ function exit_script {
 }
 
 # Install required components
-$script_path/install.sh "deploy" || exit_script "Development components installation failed"
+$script_path/install.sh "pipeline" || exit_script "Development components installation failed"
 
 # Deploy services
-$script_path/deploy.sh "deploy" || exit_script "Development of services failed"
+$script_path/deploy.sh "pipeline" || exit_script "Development of services failed"
```

Comment:
I think you meant to write "Deployment of services" rather than "Development of services"

Reply:
Done.